### ISSUE TITLE
Plans on adding webpack 4 support?

### ISSUE THREAD

**Issue by aureliusm** [tdeNL/tde-webpack-mjml-plugin#6]
Currently this error appears with webpack 4:

```
(node:29987) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
/node_modules/tde-webpack-mjml-plugin/src/webpack-mjml-store.js:68
      let response = mjml2html(contents);
                     ^

TypeError: mjml2html is not a function
    at /node_modules/tde-webpack-mjml-plugin/src/webpack-mjml-store.js:68:22
    at /node_modules/graceful-fs/graceful-fs.js:90:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:53:3)
```

Thanks!

**Comment by graycrow** [comment]
It's probably not related to Webpack 4, but to [this mjml issue](https://github.com/mjmlio/mjml/issues/1387).

To fix this, `.default` should be removed on the followin line:

https://github.com/tdeNL/tde-webpack-mjml-plugin/blob/dc6224feeb166aa213a6bfe6caf96224ce266021/src/webpack-mjml-store.js#L3

**Comment by jordikroon** [comment]
Released with 1.4.0!

**Comment by mauricesnip** [comment]
I'm still getting the deprecation warning using `"tde-webpack-mjml-plugin": "^1.4.0"`.

```(node:30059) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead```

Any pointers on how to solve this?
